sklearn/model_selection/tests/test_validation.py

```diff
@@ -972,6 +975,18 @@ def test_cross_val_predict_unbalanced():
                               decimal=12)
 
 
+def test_cross_val_predict_y_none():
+    # ensure that cross_val_predict works when y is None
+    mock_classifier = MockClassifier(allow_nd=True)
```

Comment:
`allow_nd` is not required for this test:

```suggestion
    mock_classifier = MockClassifier()
```

(So future developers do not assume `allow_nd` is required.)